src/bin/options.ts

```diff
@@ -0,0 +1,54 @@
+import commander from 'commander';
+import config from '../config';
+
+function parseNumber(v: string): number {
+  const num = parseInt(v);
+  if (isNaN(num)) {
+    throw new commander.InvalidOptionArgumentError(`${v} is not a number`);
+  }
+  return num;
+}
+
+const clientHost = new commander.Option(
+  '-ch, --client-host <address>',
+  'Client Host Address'
+)
+  .env('PK_CLIENT_HOST')
+  .default(config.defaults.clientHost);
+
+const clientPort = new commander.Option(
+  '-cp, --client-port <port>',
+  'Client Port'
+)
+  .argParser(parseNumber)
+  .env('PK_CLIENT_PORT')
+  .default(config.defaults.clientPort);
+
+const ingressHost = new commander.Option(
+  '-ih, --ingress-host <address...>',
+  'ReverseProxy ingress host'
+)
+  .env('PK_INGRESS_HOST')
+  .default(config.defaults.ingressHost);
```

Comment:
So for `--ingress-host`, it seems that we provide a list of hosts here. e.g. `pk agent start --ingress-host 127.0.0.1 ::1`. Is this correct?

I believe @CMCDragonkai's intention was to have this be able to be specified by providing the `--ingress-host` flag multiple times. e.g. `pk agent start --ingress-host 127.0.0.1 --ingress-host ::1`. Similar to `--listen` here: https://git-scm.com/docs/git-daemon#Documentation/git-daemon.txt---listenlthostoripaddrgt

This approach seems fine to me, but is this something you want changed @CMCDragonkai?

Reply:
Its a feature of the varadic option that you can specify it like `--ingress-host host1 host2` or like `--ingress-host host1 --ingress-host host2` or any combination of the two. so that is already supported.

Reply:
Great, didn't realise that both options were supported through this - just thought this supported a list of them. This can be resolved then.

Reply:
That's useful!